DOC-6457-Symbolicate-crash-logs-iOS

ibsoln opened the pull request:
https://issues.couchbase.com/browse/DOC-6457

ibsoln committed:
DOC-6457-Symbolicate-crash-logs-iOS

https://issues.couchbase.com/browse/DOC-6457

ibsoln commented:
@rajagp  @pasin  I have made the symbolication change (https://issues.couchbase.com/browse/DOC-6457).
It is staged https://ibsoln.github.io/stage/DOC-6457/couchbase-lite/2.7/swift.html#troubleshooting-3 for a review.
Can you let me know if this is ok or if I need to make changes .

jayahariv commented on modules/ROOT/pages/_partials/symbolicate-iOS.adoc:
```diff
@@ -0,0 +1,80 @@
+
+.Third party crash reporting tools
+====
+* Crashlytics legacy: https://docs.fabric.io/apple/crashlytics/missing-dsyms.html
```
As Priya suggested, we should include new Crashlytics link with legacy. 
https://firebase.google.com/docs/crashlytics/get-deobfuscated-reports

ibsoln replied:
Thanks for the timely review @jayahariv.

Since the legacy site ends in May I have replaced the original link with the Firebase Crashlytics link; see commit [9c2ee5291e27a4f1cc98af2a8749c496fb19244f](https://github.com/couchbase/docs-couchbase-lite/pull/203/commits/9c2ee5291e27a4f1cc98af2a8749c496fb19244f).

Is this good to go?

ibsoln committed:
DOC-6457-Symbolicate-crash-logs-iOS

https://issues.couchbase.com/browse/DOC-6457

Replace legacy crashlytics link crashlytics with Firebase Crashlytics link

ibsoln commented:
@rajagp, @jayahariv, @pasin can this be signed-off now? Or are changes required?

jayahariv reviewed (approved):
I will recommed others review before merging this.

ibsoln commented:
> I will recommed others review before merging this.

Thanks @jayahariv.

@couchbase/documentation-writers can I get an and-1 review on this please?
